src/breadcrumb-group/__integ__/breadcrumb-group.test.ts

```diff
@@ -107,4 +107,13 @@ describe('BreadcrumbGroup', () => {
       );
     })
   );
+
+  test(
+    'Attachs funnel name attribute to last breadcrumb item',
+    setupTest(async (page, browser) => {
+      await page.setMobileViewport();
```

Comment:
Why only test this on mobile viewport?

Reply:
I added a desktop viewport test as a unit test as its possible there. Mobile viewport can't be tested there because of the necessary css breakpoint conditions.

Reply:
Well, I guess the unit test below covers the general case. No strong opinion here.